src/components/date-picker/date-picker.st.css

```diff
@@ -22,36 +21,51 @@
     dropdownWidth: 330px;
     dropdownArrowWidth: 11px;
     dropdownArrowHeight: 12px;
-    iconWidth: 36px;
+    iconSize: 16px;
+    iconOffset: 37px;
+    inputHeight: 36px;
+}
+
+.root {
+    display: inline-block;
+    height: value(inputHeight);
```

Comment:
is root is not the input, so why set its height? (instead of it growing to the size of its children)